qt/applications/workbench/workbench/plotting/figuremanager.py

```diff
@@ -23,6 +23,7 @@
 from matplotlib._pylab_helpers import Gcf
 from matplotlib.backend_bases import FigureManagerBase
 from matplotlib.backends.backend_qt5agg import (FigureCanvasQTAgg)  # noqa
+from matplotlib.pyplot import Axes
```

Comment:
Can we avoid importing from `pyplot` and pull from `matplotlib.axes` instead? `pyplot` is a heavy import and does a lot more than just define classes.